docs/migration-guides/qiskit-backend-primitives.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Migrate to primitives for third-party provider users
+description: Migrate to primitives for users of third-party providers with `backend.run`
+
+---
+
+# Migrate to primitives for users of third-party providers with `backend.run`
+
+The `backend.run()` interface, previously the primary access point for IBM&reg; QPUs, has been superseded 
+by the Qiskit [primitives interface](/guides/primitives) in `qiskit_ibm_runtime`. 
+While this has become the standard for IBM providers, not all third-party providers have transitioned 
+to the primitives framework, and `backend.run()` remains in Qiskit for backward compatibility.
```

Comment:
Let's focus this on it being a change in standard interface defined in Qiskit, rather than a change in IBM Quantum.

```suggestion
The `backend.run` interface was the primary access point to quantum hardware and simulators offered by third party providers that integrate with Qiskit. The evolution of [user access patterns for QPUs](/migration-guides/qiskit-runtime#why-migrate-to-qiskit-runtime-primitives) has led to the Qiskit primitives (Sampler and Estimator) superseding this interface. However, not all third-party providers have transitioned to the primitives framework, and `backend.run()` remains in Qiskit for backward compatibility.
```